ytmusicapi/ytmusic.py

```diff
@@ -81,7 +83,12 @@ def __init__(self,
         self.proxies = proxies
         self.cookies = {'CONSENT': 'YES+1'}
 
-        self.headers = prepare_headers(self._session, proxies, auth)
+        input_json = load_headers_file(self.auth)
```

Comment:
Please also consider the case where self.auth is None (non-authenticated instances). Tests won't run on this changeset for this reason